apps/web/src/data/en/index.ts

```diff
@@ -806,6 +807,32 @@ export const loggedInData = {
           licence: 'Licence',
           licenceHelpText:
             'External content with the following licenses may be integrated on serlo.org:',
+          licenceFree: 'Licence Free Images',
+          pixabayText: 'Pictures will be fetched from Pixabay', // TODO: fix text
+          pixabayLoadedText: "These pictures were loaded from Pixabay's API.", // TODO: fix text
+          searching: 'Searching for images ...',
+          loadingImage: 'Downloading image ...', // "Bild wird heruntergeladen..."
+          noImagesFound: 'No images found',
+          searchTags: {
+            math: 'Mathematikunterricht',
+            chemistry: 'Chemieunterricht',
+            physics: 'Physikunterricht',
+            biology: 'Biologiediagramme',
+            tech: 'Technologie in der Schule',
+            informatik: 'Informatikkurs',
+            geometry: 'Geometrische Figuren',
+            experiments: 'Experimente',
+            science: 'Naturbeobachtungen',
+            class: 'Klassenzimmer',
+            books: 'Schulbücher',
+            hof: 'Schulhof',
+            teaching: 'Lehrmaterialien',
+            board: 'Schultafel',
+            groupWork: 'Gruppenarbeit',
+            projects: 'Schulprojekte',
+            games: 'Lernspiele',
+            creative: 'Kreatives Lernen',
```

Comment:
If you want to keep them in the i18n strings the tags need to be english here.
Also then you need to test the english and the german versions as well. 
Personally I'd prefer to move them out and only offer the feature in the german versions anyway since that is less work overall.

Reply:
Agreed with @elbotho. Let's enable it in the German version only
```
{
math: 'Mathematik',
nature: "Natur',
plants: 'Pflanzen',
adventure: 'Abenteuer',
teamwork: 'Teamwork',
journey: 'Reise",
sports: 'Sport',
chemistry: 'Chemie',
lab: "Labor',
tech: 'Technologie',
physics: 'Physik',
humans: 'Menschen',
pyramide: 'Pyramide',
cylinder: 'Zylinder',
art: 'Kunst',
music: 'Musik',
school: 'Schule'
}
```

Reply:
Moved to code, enabled only for German

Reply:
@GregorZupan is that the revised list? "Labor" does find the english Labor (arbeit) instead.
Menschen has only one hit and Physik has no hits for me.

Reply:
Menschen gets loads of hits, but unforunately Physik still gets none

Reply:
It's weird. Sometimes i get more hits sometimes less. I added more than enough words so if Physik doe not have any hit, we'll remove it